src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/_client_factory.py

```diff
@@ -5,7 +5,20 @@
 
 from azure.mgmt.batch import BatchManagementClient
 
+import azure.batch.batch_service_client as batch
+import azure.batch.batch_auth as batchauth
```

Comment:
Oh good catch - thanks!
Seems like imports are one of our major loading perf issues at this point